src/page/ConfigPageLayout/tabs/AccessTokensTab.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import { Alert, Button, Modal, TextLink } from '@grafana/ui';
+
+import { FaroEvent, reportError, reportEvent } from 'faro';
+import { useCanWriteSM } from 'hooks/useDSPermission';
+import { useSMDS } from 'hooks/useSMDS';
+import { Clipboard } from 'components/Clipboard';
+
+import { ConfigContent } from '../ConfigContent';
+
+export function AccessTokensTab() {
+  const canCreateAccessToken = useCanWriteSM();
+  const smDS = useSMDS();
+  const [showModal, setShowModal] = useState(false);
+  const [error, setError] = useState<string | undefined>();
+  const [token, setToken] = useState<string | undefined>();
+
+  const showTokenModal = async () => {
+    try {
+      reportEvent(FaroEvent.CREATE_ACCESS_TOKEN);
+      const token = await smDS.createApiToken();
+      setToken(token);
+      setShowModal(true);
+    } catch (e) {
+      const cast = e as Error;
+      reportError(cast, FaroEvent.CREATE_ACCESS_TOKEN);
+      setError(cast.message ?? 'There was an error creating a new access token');
+    }
+  };
+
+  return (
+    <ConfigContent title="Access tokens">
+      <ConfigContent.Section>
+        <h4>Synthetic monitoring</h4>
```

Comment:
Should be a h3 (well, `<Text element="h3" variant="h4">`.

Reply:
I guess that you're trying to say "Skipping heading ranks can be confusing and should be avoided". 👍🏻 
Also meaning that "Access tokens" should be in a "h2" (not a "h3").